tests/h/views/admin_admins_test.py

```diff
@@ -3,23 +3,23 @@
 from __future__ import unicode_literals
 
 import mock
-from pyramid import httpexceptions
 import pytest
+from pyramid import httpexceptions
```

Comment:
What’s your sorting algorithm here? Is is that all `from foo import bar` statements come after all `import baz` statements?